### ISSUE TITLE
Checkerboard pattern in decoder output for 256x256 input

### ISSUE THREAD

**Issue by zimonitrome** [lucidrains/unet-stylegan2#8]
Hello!
I am currently using the discriminator only in a custom GAN but I am often noticing that the decoder output produces checkerboard patterns. Even when useful information is seen, the checkerboard appears on top.

![image](https://user-images.githubusercontent.com/17656709/101456169-f1eeb380-3933-11eb-86f2-8dd03bfe4f7b.png)

**Comment by zimonitrome** [comment]
Note: This is however without CutMix regularization.

**Comment by lucidrains** [comment]
@zimonitrome are you using your own custom generator?

**Comment by zimonitrome** [comment]
@lucidrains Yes, I am using a U-Net generator as a cGAN. I tried adding both CutMix and gradient penalty (instead of previously using spectral normalization) but some networks just seem to collapse very quickly. I get that this is not really your concern since I am using a lot of custom setup 😛 but I was interested if you had noticed something similar with larger resolutions.
